stylesheet.css

```diff
@@ -0,0 +1,4 @@
+body {
+    background-color: black;
+    color: gold;
+}
```

Comment:
Let's add basic styling with the following requirements:
- The form should be centred
- The form submit button should be styled
- Form sections like `gender`, `name` etc should be styled and group accordingly.

Use this site https://mint.intuit.com/login.event?task=S is an inspiration for the design. Note that your style does not necessarily have to be as styled as the suggested link provided.